README.md

```diff
@@ -152,6 +144,12 @@ if (QUnit) {
  JsReporters.TapReporter.init(runner);
 ```
 
+## API
+
+**autoRegister()**
```

Comment:
Usually a usage example is pretty much required. In this case, pointing out the example above would be fine.